resources/js/src/app/components/basket/AddToBasket.vue

```diff
@@ -187,7 +187,8 @@ export default {
                 !this.hasChildren &&
                 !(this.minimumQuantity != 1 || this.intervalQuantity != 1) &&
                 !this.requiresProperties &&
-                this.hasPrice;
+                this.hasPrice &&
+                !this.isSet;
```

Comment:
isSet gibt es nur im SingleItem, da es auf dessen VueX Module zugreift. Am besten gibst du ob es set ist über einen Parameter in die komponente, ähnlich wie das mit isSalable im Makro gelöst ist

Reply:
Done.